tests/Backend/Workspaces/MetadataFromExasolWorkspaceTest.php

```diff
@@ -233,89 +229,6 @@ public function testCopyImport(): void
         $this->assertEquals([], $table['columnMetadata']);
     }
 
-    public function testMetadataManipulationRestrictions(): void
```

Comment:
testMetadataManipulationRestrictions sem smazal ten test testoval, že když mám feature tables-defintion tak nejde aktualizovat storage metadata, to sa změnilo a teď nejde aktualizovat storage-metadata pokud má tabulka typy. Jako next bych uvažoval o smazání storage-types protože to teď dělá runner.

Reply:
No, ale na to, že nejdou upravovat metadata, když má tabulka typy jsi test nepřidal, ne? A to bys IMHO měl.

Reply:
Nebo se to testuje jinde?

Reply:
Přidal sem test do common.